### ISSUE TITLE
`gem "neat", :git => "git://github.com/thoughtbot/neat.git", :branch => "neat-2.0.0"` breaks bourbon and does not work

### ISSUE THREAD

**Issue by whmii** [thoughtbot/neat#460]
Was working on a middleman app this weekend and when I install neat-2.0.0, I get the following error.

<img width="1395" alt="screen shot 2016-07-09 at 2 09 49 pm" src="https://cloud.githubusercontent.com/assets/2489247/16709434/d616f462-45de-11e6-9d28-b959a7631a2d.png">

**Comment by whmii** [comment]
gemfile: 

```ruby
# If you do not have OpenSSL installed, update
# the following line to use "http://" instead
source 'https://rubygems.org'

gem "middleman", "~>3.4.0"
gem "bourbon", "5.0.0.beta.6"
gem "neat", :git => "git://github.com/thoughtbot/neat.git", :branch => "neat-2.0.0"

# Live-reloading plugin
gem "middleman-livereload", "~> 3.1.0"

# For faster file watcher updates on Windows:
gem "wdm", "~> 0.1.0", :platforms => [:mswin, :mingw]

# Windows does not come with time zone data
gem "tzinfo-data", platforms: [:mswin, :mingw, :jruby]

```

**Comment by whmii** [comment]
After further inspection, The way bourbon and neat add to `SASS_PATH` works if there is 1 lib using this method but not if there are multiple.

```
require "bourbon/engine"

bourbon_path = File.expand_path("../../core", __FILE__)
ENV["SASS_PATH"] = [
  ENV["SASS_PATH"],
  bourbon_path,
].compact.join(File::PATH_SEPARATOR)
```

**Comment by tysongach** [comment]
Fixed in 4af34f2dfe1f23bcb2ae737fdd137765e6eab96c.
